StepDefinitions/MarsLanguageSkillsTestsStepDefinitions.cs

```diff
@@ -0,0 +1,115 @@
+using Mars_Language_Skills.Pages;
+using MarsLanguageSkillsSpecflow.Pages;
+using MarsLanguageSkillsSpecflow.Utilities;
+using OpenQA.Selenium.Chrome;
+
+namespace MarsLanguageSkillsSpecflow.StepDefinitions
+{
+    [Binding]
+    public class MarsLanguageSkillsTestsStepDefinitions : CommonDriver
+    {
+        LoginPage loginPageObj = new LoginPage();
+        ProfilePage profilePageObj = new ProfilePage();
+        LanguageFeature languageFeatureObj = new LanguageFeature();
+        SkillFeature SkillFeatureObj = new SkillFeature();
+
+
+        [Given(@"User logs in Mars portal")]
+        public void GivenUserLogsInMarsPortal()
+        {
+            // IWebDriver driver = new ChromeDriver();
+            driver = new ChromeDriver();
+
+            //LoginPage obeject initialization
+
+           // LoginPage loginPageObj = new LoginPage();
+            loginPageObj.LoginActions(driver, "krits17.kt@gmail.com", "Singapore_24");
+           
+        }
+
+        [When(@"User navigates to profile page")]
+        public void WhenUserNavigatesToProfilePage()
+        {
+            profilePageObj.VerifyLoggedinUser(driver);
+        }
+
+        [When(@"User adds a language with level, and clicks Add button")]
+        public void WhenUserAddsALanguageWithLevelAndClicksAddButton()
+        {
+            languageFeatureObj.AddLanguages(driver);
+        }
+
+        [Then(@"User should see the language and it's level added to the profile")]
+        public void ThenUserShouldSeeTheLanguageAndItsLevelAddedToTheProfile()
+        {
+           languageFeatureObj.VerifyLanguagesAdded(driver);
```

Comment:
please do not pass the driver in action methods instead extend to common driver or pass via constructor upon creation of objects, please change this in all action methods!